docs/source/markdown/podman-farm.1.md

```diff
@@ -11,6 +11,8 @@ Farm out builds to machines running Podman for different architectures.
 
 Manage farms by creating, updating, and removing them.
 
+Note: All farm machines must have an updated podman version of at least v4.9.0.
```

Comment:
Recommend replacing "updated podman version of at least" with "minimum Podman version of"

Reply:
Fixed